### DISCUSSION TITLE
Google search display domain name insted of my title

### DISCUSSION THREAD

**Discussion by KP-MobileTechie** [vercel/community#5744] [Help]
### Summary

I have upload my website using the vercel on the google search console but on the google search show me the main title is domain name vercel and I want to replace that with my name could you help me.

### Steps to Reproduce

I have give the title to my website and live in google search console but after the live a get that issue and I have tried many solutions but anything not work

**Comment by guptaashwanee** [comment]
Google displays the title based on the tags available in the head section of the website, If nothing is available, the domain often is taken. 

- <title> elements 
- Heading \<h1\> elements,
- og:site_name

You can also check this docs link: https://developers.google.com/search/docs/appearance/site-names

**Comment by KP-MobileTechie** [comment]
Yes I already do that you can check my website code in this https://kp-mobiletechie.vercel.app and inspect the elements and see I have tried all possible solution but anything is not worked for me.

**Reply by endosama** [reply]
Having the exact same issue and all parameters specified here are in the website and in the html crawled by the GoogleBot.
Currently using next@14.0.4 👍

[a comment by an automated account is not shown]
